cmd/dhcpserver/get.go

```diff
@@ -47,12 +47,22 @@ var getCmd = &cobra.Command{
 			return err
 		}
 
-		servers, err := pvmclient.DHCPClient.Get(id)
+		server, err := pvmclient.DHCPClient.Get(id)
 		if err != nil {
 			return fmt.Errorf("failed to get a dhcpserver, err: %v", err)
 		}
-
-		spew.Dump(servers)
+		if server.Network.Name == nil {
```

Comment:
I think it should be other way around instead of checking Network name is nil, we should check directly status of server.

Reply:
Yes, we could take that approach as well. 
Just thought that, at some point, when the Network name is available first, while the DHCP status may still report as BUILD with the current check. 
If we adopt to checking the status, we may only get to see the information only when the Status is ACTIVE. 

Just wanted to check if we want to go by status, than the available field.

Thanks!

Reply:
I dont think so that is possible, Without being DHCP server being active having a network, Never the less if you want to keep this behavior, then the log message should be change to explicilty call out the DHCP server status rather than hard coding it to ACTIVE

Reply:
cool, May be a second before it becomes active.